---
title: 2019 Jastro Research Proposal
---

## Determining the plant targets of serine protease effectors from the xylem-colonizing bacterial pathogen of tomato, __Clavibacter michiganensis__

## **Background and rationale:**

## Bacterial canker of tomato caused by __Clavibacter michiganensis____ __is a devastating disease to the tomato production industry. In California, the major producer of processing tomatoes in the United States, there have been economic losses due to outbreaks of the disease in 2015, 2016, and 2018. With tomato production value toping over a billion dollars, the steady production of this crop is imperative. Adding to the losses, there are no current preventative treatments to treat infected plants. This is due to, in part, a poor understanding of the mechanisms employed by __C____. ____m____ichiganensis____ __to cause disease. Current recommendations, including sterilizing seeds and good horticultural practices, have reduced outbreaks. However, as exemplified by the recent outbreaks in California, an improved understanding of the mechanisms of pathogenicity is necessary to enable the development of disease prevention technologies.

## __C. michiganensis____ __is a Gram-positive actinobacterium which can colonize the xylem of tomato leading to unilateral wilting, necrosis, and canker formation. The bacterium carries virulence genes necessary for causing disease on two plasmids, pCM1 and pCM2, and in a large ~129kb pathogenicity island (PAI), __Chp____/____TomA____ ______(Eichenlaub and Gartemann, 2011; Thapa __et al.__, 2017)______.__ The __Chp____ __region__ __encodes carries 17 conserved serine protease effectors. Serine proteases encoded in this region are known to be secreted in the xylem (Qingyang Liu, unpublished). Therefore, these effectors are hypothesized to inhibit host defenses by cleaving key plant protein targets in the xylem that will facilitate bacterial colonization and eventually disease (Nandi __et al.__, 2018). We will therefore systemically test individual __C. michiganensis__ proteases to determine their plant targets in tomato. From this work, we expect to elucidate a key mechanism of pathogenicity for __C. michiganensi____s__.

## **Objectives and Procedures**

## **Objective 1. ****Express and catalytically characterize**** ****C. michiganensis**** ****serine protease ****e****ffectors****.**

## In order to determine the targets of individual proteases, we will use the yeast (__Pichia ____pastoris__) expression system to express and purify recombinant protein. A total of 17 serine proteases in the Chp, Ppa, and Sbt family from the reference strain NCPPB 382 will be cloned into the expression vector pPIC9H. Other divergent serine proteases from other isolates of __C. michiganensis____ __and/or phytopathogenic __Clavibacter __species will also be cloned. All constructs will be transformed into __E. coli __strain DH5α, sanger sequenced, and transformed into __Pichia __strain GS115. Yeast colonies carrying the recombined-plasmid will be selected through a His-lacking medium and confirmed for correct recombination by re-plating the colonies onto a methanol minimum medium. One mutant for each serine protease will be grown via fermentation, expression induced by methanol, and affinity purified using nickel affinity chromatography. Purified protein will be assessed for purity by western blot, quantified via a Bradford assay, and confirmed for catalytic ability using the general substrate azocasein (Charney and Tomarelli, 1947). We expect a subset of proteases will be cloned, expressed, and protein purified within 9 months.

## **Objective 2. Identify and ****characterize ****plant targets ****using TAILS Mass Spectrometry. **

## __C. michiganensis____ __is a xylem-colonizing plant pathogen. Previous work has shown that serine protease effectors present in the __Chp____ __region are expressed in xylem-mimicking conditions (Qingyang Liu, unpublished). Therefore, we expect the plant targets of these proteases to reside within the xylem. To identify effector targets, we will collect xylem sap from about 4-week-old tomato plants and use terminal amine isotropic labeling of substrates (TAILS) mass spectrometry for each serine protease (Doucet __et al.__, 2011). Since proteolysis involves generating new N- and C-termini not originally present, TAILS utilizes isotopic labeling of terminal amine residues to identify peptides in a proteome which have undergone protease cleavage. Mass spectrometry spectra will be compared between protease treated and nontreated xylem sap. To confirm targets, non-functional catalytic mutants of each serine protease will be developed, and the mass spectroscopy spectra will be compared to the previous two spectra. We will select protease effectors for TAILS based on their conservation across all __C. michiganensis __strains and virulence promoting activates. We expect to complete this portion of the project within one to two years.

## **Qualifications:**

## I have considerable experience in molecular biology, especially working with Gram-positive plant pathogens and plant-associated bacteria. Previously, I investigated a Gram-positive actinobacteria __Rhodococcus __by characterizing their interactions with their plant hosts (Savory __et al.__, 2017). I have already developed pPIC9-6xHis-__c____hpC__ expression construct in the yeast __Pichia____ __strain GS115 and am in the process of purifying recombinant protein. We will collaborate with Professor Ioannis Stergiopolous’ group and the proteomics core to complete these experiments. The Stergiopoulos lab has experience expressing effectors in __Pichia____ __and members of the Coaker lab have extensive experience in proteomics including protein expression and mass spectrometry. Finally, I am in communication with and will collaborate with the UC Davis proteomics core (Brett Phinney). I will be working with a variety of personnel from each lab and the core to carry out this work.

## 

## **References:**

## **Charney, J. and ****Tomarellia****, R.M**. (1947). A colorimetric method for the determination of the proteolytic activity of duodenal juice. __J. Biol. Chem. __**171**, 501-505.

## ******Doucet, A., ****Kleifeld****, O., ****Kizhakkedathu****, J.N. and Overall, C.M.** (2011) Gel-Free Proteomics, Methods and Protocols. __Methods in molecular biology (Clifton, N.J.)__ **753**, 273–287.

## ******Eichenlaub****, R. and ****Gartemann****, K.-H.** (2011) The Clavibacter michiganensis Subspecies: Molecular Investigation of Gram-Positive Bacterial Plant Pathogens. __Phytopathology__ **49**, 445–464.

## ******Nandi, M., Macdonald, J., Liu, P., ****Weselowski****, B. and Yuan, Z.** (2018) Clavibacter michiganensis ssp. michiganensis: bacterial canker of tomato, molecular interactions and disease management. __Mol Plant ____Pathol__ **19**, 2036–2050.

## ******Savory, E.A., Fuller, S.L., Weisberg, A.J., et al.** (2017) Evolutionary transitions between beneficial and phytopathogenic Rhodococcus challenge disease management. __Elife__ **6**, e30925.

## ******Thapa, S.P., ****Pattathil****, S., Hahn, M., Jacques, M.-A., Gilbertson, B. and Coaker, G.** (2017) Genomic analysis of Clavibacter michiganensis reveals insight into virulence strategies and genetic diversity of a Gram-positive bacterial pathogen. __Mol Plant Microbe In__.

## 
