---
title: Beyond pathogens: microbiota interactions with the plant immune system
---

## **Authors**: [[Paulo Jose PL Teixeira]], [[Nicholas R Colanianni]], [[Connor R Fitzpatrick]], and [[Jeffrey L Dangl]]

## **Journal**: [[Current Opinion in Microbiology]]

## **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:adc2b0cd-6a25-490f-8be8-061a8ccfdea5

## **Tags**: #[[plant immunity]] #[[PRR-Triggered Immunity]] #PRRs #[[Beneficial bacteria]]

## **Abstract**:
### much less is known about plant immunity in the context of the wealth of commensals that inhabit plants

### Recent research indicates that, just like pathogens, commensals in the 
plant microbiome can suppress or evade host immune responses.

## **Introduction**:
### Briefly, plants possess receptors that recognize non-self or modified-self molecules which indicate the presence of potential invaders

### adapted pathogens have evolved effector molecules to interfere with MTI and host physiology

### most microbes with which plants interact are non-pathogenic, yet many of them express molecules that are potentially recognized by the plant immune system

### one major question of plant microbiome research is whether and how the 
plant immune system distinguishes commensals from pathogens during 
microbiome assembly

## **Evidence for the participation of the plant immune system in microbiome assembly**:
### have adapted ways to distinguish their evolutionary partners from other microorganisms

### the assembly of at least some of the plant microbiome may represent mere niche filling

### some fraction of the commensal community has adaptive value to the host

### commensal and pathogenic Pseudomonas strains, the transition between 
[[lifestyles]] is based on the gain/loss of only a very few [[pathogenicity island]]

### healthy Arabidopsis roots and leaves revealed that 608 bacteria (97%) have the potential to produce putatively immunogenic [[MAMPs]]

### PTI response shown to repress the growth of pathogens and is potentially activated in response to commensals as well

## 
### ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2F3SiZl7BLSn.png?alt=media&token=73e8edcc-fbbf-41ec-be05-e8c3875f8299)
**Commensal MAMP profiles and potential mechanism used by microorganisms to evade the plant immune system.** (a) Bacteria isolated from healthy Arabidopsis plants harbor potentially immunogenic [[MAMPs]] while others show large divergence from the canonical, which may contribute to evasion of the plant immune system. The tree includes 627 Arabidopsis-derived isolates and was generated by pruning the 3837 microbial tree in Levy et al. [29• ]. MAMP genes were identified using custom built profile hidden Markov models and the MAMPs were identified after aligning all MAMP genes with MUSCLE [105,106]. Percent identity was calculated using edit distance. The tree was generated with iTOL [107]. The canonical sequences are from Micrococcus lysodeikticus ([[csp22]]) [108], Escherichia coli (elf18) [109], Pseudomonas syringae pv tabaci (flg22) [36], and Phytophthora parasitica (nlp20) [110]. MAMP divergence could lead to plant immune evasion by (b) sequence variation, (c) MAMP degradation or sequestration, or (d) MAMP modification as detailed in the text.


### downstream [[PRR-Triggered Immunity]] signaling exhibit reduced defense against an avirulent mutant of a normally pathogenic [[Pseudomonas syringae]] strain and are unable to maintain normal leaf endophytic bacterial communities under high humidity
#### Inability to regulate the growth of [[endophytic]] bacterial communities led to mild chlorosis and necrosis in some leaves, resembling dysbiosis

### 

## **Strategies used by the microbiome to evade or suppress plant immunity**:
### Plant immune receptors do not distinguish between microbial [[lifestyles]] 
and recognize ligands that can be present in both pathogens and 
commensals

### PRR-Triggered Immunity suppression is used by pathogenic microbes to bypass the plant immune system but has also recently been reported for non-pathogenic microbes
#### beneficial rhizobacterium Pseudomonas simiae WCS417 promotes plant 
growth and suppresses part of the transcriptional response that is 
triggered by the bacterial MAMP flg22

### [[Endophytes]] can also prevent MAMP-triggered cytosolic calcium influx in Arabidopsis
#### A recent study found that the plant growth-promoting bacterium Pseudomonas capeferrum WCS358 produces organic acids that lower the 
extracellular pH and interfere with the response to flg22

### Microbes have evolved at least 3 mechanisms to evade [[PRR-Triggered Immunity]]:
#### 1. MAMP divergence
2. MAMP degradation/sequestration
3. MAMP modification

## 
### **MAMP divergence**:

### Microbes might evade [[PRR-Triggered Immunity]] by evolving MAMP variants that no longer bind to or activate the corresponding plant PRR 

### counter-adaptive, since alteration of [[MAMPs]] sequences and structures may 
impair the positive function of the microbial MAMP-containing molecule
#### some flg22 variants that lose immunogenicity also lose motility

#### distribution is likely to be MAMP-dependent

#### while less than 1% of elf18 variants identified diverge from the canonical epitope by at least 50%

#### differential expression stringencies: elf18 receptor EFR is not expressed in Arabidopsis roots, while FLS2 is

### Arabidopsis and tomato lines display large variation in their response 
to different MAMPs and even to the same MAMP variant, indicating that 
MAMP recognition across plant populations is evolving

## 
### **MAMP degradation/sequestration**:
#### Microbes have evolved proteases that digest their MAMPs or proteins that sequester MAMPs to hide them from plant receptors
##### [[Pseudomonas syringae]] DC3000 secretes a protease that, through the degradation of flagellin, decreased [[PRR-Triggered Immunity]] and increased this strain’s growth in Arabidopsis and tomato leaves

##### Several fungal pathogens have evolved chitin-binding proteins ([[LysM]] /inactive chitianases) that sequester free chitin fragments to prevent the activation of plant [[PRRs]]

## 
### **MAMP modification**:
#### [[Nicotiana benthamiana]] secretes [[glycosidases]] that strip the glycan shield from the bacterial flagellum, allowing plant proteases to release the 
immunogenic [[flg22]] peptide for recognition by the [[FLS2]] receptor

#### pathogens can evade [[flg22]] recognition by either inhibiting plant 
glycosidases or by modifying the glycan moieties that cover their flagellum

## 

## **The microbiome functions as an extension of the plant immune system**:
### Importantly, this seems to be largely determined by only two main 
mechanisms thus far:
#### 1. direct microbe–microbe interactions
2. stimulation or priming of the plant immune system 

### [[Microbe–microbe interactions]] play an increasingly evident role in the suppression of pathogens and can serve as a first line of defense against invading organisms in plants
#### a molecule secreted by the Pseudomonas piscium ZJU60 strain, which was 
isolated from infected wheat head, antagonizes the fungus Fusarium  graminearum by inhibiting one of its histone acetyltransferases 

### Sphingomonas melonis strain isolated from Arabidopsis activates a subset of plant defense genes and promotes immunity against the bacterial pathogen [[Pseudomonas syringae]] DC300
#### This protection is lost in the bak1/bkk1 mutant, indicating that this 
commensal likely stimulates plant immunity through MAMP recognition by 
[[PRRs]] that rely on [[BAK1]] as a [[co-receptor]]

## 

## **Context-dependent immune modulation shapes the plant microbiome across environments**:
### Plant responses to environmental variation are interconnected with plant immunity

### Environmental variation can modulate plant immunity through a number of non-exclusive mechanisms
#### 1. different environments can directly modulate the expression of plant immune output
##### under increased temperature several NLR receptors lose function

#### 2. hormonal regulators of plant responses to abiotic stress have an antagonistic effect on plant immunity through the suppression of the jasmonic acid (JA) and salicylic acid (SA) defense pathways
##### such effects are so potent that microbial pathogens hijack this interplay to facilitate their invasion

#### 3. shared signaling components can jointly coordinate plant responses to abiotic stress and plant immunity
##### the [[PRR]] chitin elicitor receptor kinase 1 ([[CERK1]]), is responsible for mounting a plant defense against fungal pathogens but also strongly regulates the expression of genes required for salt stress
