---
title: Comparative Genomics and Phylogenetic Analyses Suggest Several Novel Species within the Genus Clavibacter, Including Nonpathogenic Tomato-Associated Strains
---

- **Authors**: [[Ebrahim Osdaghi]], [[Touraj Rahimi]], [[S Mohsen Taghavi]], [[Maryam Ansari]], [[Sadegh Zarei]], [[Perrine Portier]], [[Martial Briand]], [[Marie-Agnes Jacques]]
- **Journal**: [[Applied and Environmental Microbiology]]
- **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:638cfb37-93ac-4bc7-a234-5146c2d93d02
- **Tags**: #clavibacter #[[Comparative Genomics]]
- **Abstract**:
	- current taxonomic descriptions of the genus do not reﬂect the existing diversity of the strains
	- three new subspecies (i.e., C. michiganensis subsp. californiensis, C. 
	  michiganensis subsp. chilensis, and C. michiganensis subsp. phaseoli)
	- a number of pathogenicity-related genes were noted to be present in all 
	  the strains regardless of their pathogenicity characteristics
- **Introduction**:
	- the presence of nonpathogenic strains in commercial seeds or propagative parts of plants will interfere in the accurate detection of the pathogens
	- all the tomato- and pepper-associated Clavibacter sp. strains were classiﬁed as members of C. michiganensis subsp. michiganensis regardless of whether they were pathogenic or nonpathogenic on the host of isolation
	- since false-positive results would lead to the rejection of seed/ seedling lots on an economically signiﬁcant scale
	- strains associated with solanaceous vegetables contain a large fraction 
	  of diversity within the Clavibacter sp. members, and much of the 
	  molecular, phylogenetic, and genomic information for these strains 
	  remains unexplored
	- Draft genome sequence-based phylogenetic analyses revealed a higher diversity among the nonpathogenic strains of Clavibacter spp. than has  previously been reported
	- C. michiganensis subsp. chilensis and C. michiganensis subsp. phaseoli need to be considered members of one species, according to the 99% genome similarity among the type strains
- **Results**:
	- pepper plants inoculated using the same strain showed only stem canker symptoms on the site of inoculation, with no wilting or plant death ([[Clavibacter host range]])
	- However, neither C. michiganensis subsp. phaseoli nor C. michiganensis 
	  subsp. chilensis induced symptoms on the inoculated plant species, i.e.,
	   common bean, cowpea, pepper, mung bean, and tomato
	- This could be an indication of the fact that C. michiganensis subsp. phaseoli and C. michiganensis subsp. chilensis were unable to endophytically colonize
	- CFBP 8615 and CFBP 8616, although no symptoms were observed on common bean, cowpea, pepper, mung bean, and tomato plants, bacterial colonies similar to those originally inoculated were consistently reisolated from the leaf tissues of common bean cultivar Navy plants inoculated with 
	  CFBP 8616
	- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FP_zwImWQZ1.png?alt=media&token=37bcb1d2-54ed-44b2-9761-91121bb58fcd)
-
	- given the differences in their pathogenicities and biochemical characteristics as well as the belowthreshold genomic similarity, the two taxa C. michiganensis sensu stricto and C. michiganensis subsp. californiensis could be considered separate species
	- [[Clavibacter sepedonicus]] strains formed a monophyletic cluster separate from all the other lineages by ANI values of < 93%
	- [[Clavibacter insidiosus]] and [[Clavibacter nebraskensis]] strains clustered in a monophyletic group showing 95% ANI between the type strains of the species
	- While strain [[CFBP 7494]] showed only a 64.60% dDDH value with the type strain of [[Clavibacter insidiosus]], the 96% ANI (on the upper edge of species deﬁnition) prevents differentiation of this strain from the C. insidiosus species.
	- [[CFBP 7494]] was isolated from tomato but was nonpathogenic on this plant species, while it has been shown that it induces disease symptoms on wheat plants under greenhouse conditions
	- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2Fv7rYeKp0ZU.png?alt=media&token=3e0c120a-6735-44bf-aefe-8e52044cd9b9)
	  **Average nucleotide identity** (ANI)-based neighbor-joining phylogenetic 
	  tree of 40 Clavibacter sp. strains constructed using the ANI calculator 
	  online service.
	- Strain [[CF11]], isolated from soil in a tomato-growing greenhouse (23), as well as the type strain of the pepper pathogen [[Clavibacter capsici]] (PF008T) clustered in a monophyletic clade, while they differed from one another with 95% ANI and 58.50% dDDH values
	- The type strain of [[Clavibacter tessellarius]] showed 93% ANI with the type strains of all the other subspecies/species, confirming the wheat pathogen as a stand-alone species
	- However, neither strain [[CFBP 8017]] nor strain [[DOAB 609,]] which were clustered in a shared clade with the type strain of [[Clavibacter tessellarius]], could be included within this species.
	- The ANIs of CFBP 8017 and DOAB 609 with the type strain of [[Clavibacter tessellarius]] were 95% and 93%, respectively, while the dDDH values between the same strains were 57.70% and 49.00%, respectively
	- lack of pathogenicity determinant genes/clusters in all the tomato-associated nonpathogenic strains evaluated in this study
		- For the chp gene cluster (i.e., loci CMM_0034 to CMM_0077 in the [[NCPPB  382]] genome sequence [accession no. AM711867.1]), only a fraction of the genes were detected in the nonpathogenic strains
		- A serine protease ([[ppaD]] - CMM_0075) and a putative ATPase (CMM_0067) were found only in CASJ009 and [[CFBP 7491]]
		- Among the pathogenicity determinant genes inside the chp gene cluster, ppaA (CMM_0041), pelA1 (CMM_0043), pelA2 (CMM_0051), chpC (CMM_0052), and [[chpG]] (CMM_0059) were found in none of the evaluated nonpathogenic strains.
		- Non-chp pathogenicity determinant genes clvG (CMM_1963), clvF (CMM_1964), clvA (micA - CMM_1967), and perF (CMM_2382) were also not detected in the evaluated nonpathogenic bacterial strains
		- a subtilisin-like serine protease (sbtA - CMM_0070) was found in a number of nonpathogenic strains (Table 5), while nucleotides 1 to 600 were missing in all the nonpathogenic members. The [[expansin]] encoding gene expA (CMM_1480) was found in CFBP 8017, DOAB 609, CFBP 7493, and LMG26808.
		- [[celB]] (CMM_2443), pelA1 (CMM_0043), pelA2 (CMM_0051), xysA (CMM_1673), xysB (CMM_ 1674), CMM_2691, CMM_2692, and CMM_2871, which are responsible for cell wall degradation at the later stages of tomato infection by C. michiganensis sensu stricto
		- polygalacturonase-encoding locus CMM_2871 was found in tomato-associated strains CFBP 7493 and LMG 26808, as well as the type strains of C. insidiosus (LMG 3663T), C. nebraskensis (NCPPB 2581T), and C. tessellarius (ATCC 33566T)
		- [[celB]] (CMM_2443), which is a homolog of the plasmid-born [[celA]] gene, was present in tomato-associated nonpathogenic strains CFBP 8017, DOAB 609, CFBP 7493, LMG 26808, and CFBP 7494
		- virulence-associated transcriptional regulator genes [[vatr1]] (CMM_2645) and [[vatr2]] (CMM_2969), which regulate C. michiganensis sensu stricto virulence during infection, were present in all the strains evaluated in this study regardless of their pathogenicity status
		- type strains of C. michiganensis subsp. phaseoli and C. michiganensis subsp. chilensis, as well as the two peach-colored strains CFBP 8615 and CFBP 8616, did not carry any detectable plasmid
-
- **Discussion**:
	- BLAST-based [[Comparative Genomics]] revealed that several genes (i.e., [[vatr1]], [[vatr2]], xysA, xysB, and srtA) which had previously been identiﬁed as pathogenicity determinants were present in all the pathogenic and  nonpathogenic tomato-associated strains, indicating further complexities in the functions of these genes
	-
	- The type strain of C. michiganensis subsp. phaseoli ([[CFBP 8627]]$$^T$$) was isolated from common bean seeds in Spain and reported to cause bacterial bean leaf-yellowing disease in greenhouse assays
	-
	- Further ﬁeld surveys are needed to decipher potential natural occurrence under the ﬁeld conditions and the putative frequency and prevalence of the common bean-associated Clavibacter sp. strains.