---
title: Cytosine Deaminase as a Negative Selection Marker for Gene Disruption and Replacement in the Genus Streptomyces and Other Actinobacteria
---

- **Authors**: Marie-Pierre Dubeau,1‡ Mariana Gabriela Ghinet,1‡ Pierre-Étienne 
  Jacques,2 Nancy Clermont,1 Carole Beaulieu,1 and Ryszard Brzezinski
- **Journal**: [[Applied and Environmental Microbiology]]
- **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:52f4b7fc-d7fa-44fa-a9a1-cc30ef6ec7d0
- **Tags**:
- **Abstract**:
- **Introduction**:
	- [[cytosine deaminase]] catalyzes the deamination of cytosine to uracil as a part of the [[pyrimidine salvage pathway]]
	- Counterselection markers are valuable tools facilitating gene disruption based on homologous recombination
	- The rpsL gene (encoding the ribosomal protein S12) confers a dominant streptomycin-sensitive phenotype in a streptomycin resistance background, the latter being obtained by spontaneous mutations of the rpsL gene
	- most gene disruption procedures include a laborious colony replication step to obtain isolates that have lost an antibiotic resistance marker.
	- A previous study of pyrimidine salvage pathways in Streptomyces (15) noted the absence of [[cytosine deaminase]]
	- no apparent [codA]([[cytosine deaminase]]) orthologs in genomic sequences of most actinobacteria
- **Results**:
	- **Construction of genes for selection.**
		- For positive selection, we used the aminoglycoside resistance gene from pFDNeo-S, providing this gene with the promoter from the aacC4 gene
	- **Vector construction.**
		- Both polylinkers will accommodate DNA fragments in insertional gene disruption procedures, while only the polylinker XbaI-PvuI will be used for cloning the fragments involved in gene replacement, including in-frame deletion fragment
		- Replacing codA(s) by the mutated version codA(sm), we obtained pMG201M (see Fig. S1 in the supplemental material).
	- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FCDNqcijdUv.png?alt=media&token=672bd9cd-1b96-4f51-9c91-830d56776cee)
		- FIG. 1. Vector pMP201 and derivatives. (A) Map of the pMP201 vector with
		   polylinkers XbaI-PvuI and HindIII-EcoRI. neoA, aminoglycoside 
		  resistance gene with the promoter from the aacC4 gene; codA(s), 
		  synthetic version of the cytosine deaminase gene; ori, origin of 
		  replication for E. coli. The positions of oriT (origin of conjugative 
		  transfer), present in pMP301 and pMP302, and bla (ampicillin resistance 
		  gene), present in pMP302, are also shown. Note that the PvuI site in the
		   polylinker is not unique in derivatives containing the bla gene. (B) 
		  DNA sequences of polylinkers in pMP201 and derivatives (coordinates in 
		  both panels are from the pMP201 sequence, and those in panel B 
		  correspond to the first boldface base pair for each restriction enzyme).
		   Boldface letters and dashes indicate the restriction sites and the 
		  respective cleavage sites
	- Resistance to 5FC is frequent among actinobacteria.
		- The absence of cytosine deaminase, observed in S. coelicolor and Streptomyces griseus (15), seems to be frequent and is perhaps generalized among actinobacteria
	- codA(s)-based counterselection
		- Disruption cassettes amplified from the S. lividans genome were cloned 
		  into pMP302, and E. coli ET12567 (pUZ8002) was then transformed with the
		   resulting plasmid
		- In the counters-election step, the plating of 20,000 spores onto MM 
		  with 200gof 5FC ml1yielded 18 resistant colonies, resulting in a 
		  second-crossover frequency of103.
		- A  Km r colony  was picked up, and its spores were plated onto MM with Km (250  gml  1 )  and  5FC  (50  gml  1 ).
		- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FdnqZ40nmFh.png?alt=media&token=19a37e63-c329-4301-93b9-996a55076565)
			- FIG.  2.  Inactivation  of  anS. lividans SCO2657  homolog  gene(2657h)  
			  via  replacement  by  an  in-frame  deletion.  (A)  Map  of  the pMP302  
			  vector  derivative  (pMP302-2657h)  harboring  PCR-amplified S. 
			  lividans genome fragments for homologous recombination. Positions  of  
			  restriction  sites  are  listed  in  parentheses.  (B)  Diagrams showing 
			   the  wild-type  (WT)  genomic  section  ofS. lividans and  
			  its configuration after the second crossing over. I, strain with a 
			  reversion to the wild-type configuration after two recombination events 
			  involving the same PCR-amplified fragment; II, strain with an in-frame 
			  deletion obtained by recombination events occurring alternatively in both
			   amplified segments; 2656h to 2660h,S. lividans homologs of the 
			  respectiveS. coelicolor A3(2) genes; arrows, DNA fragment used as a probe
			   inSouthern blotting. Positions of restriction sites are given. (C) 
			  Southern blotting analysis of the 2657h gene area before (WT) and after 
			  (I or II)selection  steps.  Genomic  DNA  from  the  host  strain  (WT) 
			   and  two PCR-preselected clones (I and II) resistant to 5FC was digested
			   with BglII and analyzed by Southern blotting. M, molecular size markers.
- **Discussion**:
	- Conclusion. The cytosine deaminase gene can be efficiently used as a 
	  negative selection marker for gene disruption and replacement in 
	  Streptomyces. The method can be used for any actinobacterial species 
	  naturally resistant to 5FC while sensitive to 5FU. It should be a 
	  valuable addition to the actinobacterial genetic toolbox
- Other Notes