---
title: Evolution of the U.S. Biological Select Agent Rathayibacter toxicus
---

## **Authors**: [[Edward W. Davis II]], [[Javier F. Tabima]], [[Alexandra J. Weisberg]], [[Lucas Dantas Lopes]], [[Michele S. Wiseman]], [[Michael S. Wiseman]], [[Tal Pupko]], [[Michael S. Belcher]], [[Aaron J. Sechler]], [[Matt A. Tancos]], [[Brenda K. Schroeder]], [[Timothy D. Murray]], [[Douglas G. Luster]], [[William L. Schneider]], [[Elizabeth E. Rogers]], [[Fernando D. Andreote]], [[Niklaus J. Grünwald]], [[Melodie L.  Putnam]], [[Jeff H. Chang]]

## **Journal**: [[mBio]]

## **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:c6d9f31c-98a1-46fe-b70a-3000ee41afe6

## **Tags**: #genomics #Rathayibacter

## **Abstract**:
### A phylogenomic approach was employed to model the evolution of [[Rathayibacter]] toxicus to explain the low genetic diversity observed among isolates collected during a 30-year period of sampling in three regions of Australia

### Analyses of a data set of more than 100 sequenced [[Rathayibacter]] genomes indicated nine species-level groups.

### evidence suggested that this species experienced a dramatic event in its evolution. Its genome is signiﬁcantly reduced in size but is colinear to those of sister species.

### [[Rathayibacter]] toxicus has low intergroup genomic diversity and almost no intragroup genomic diversity between ecologically separated isolates

## Rathayibacter toxicus is a species of corynetoxin-producing [[Gram-positive pathogen]] that causes annual ryegrass toxicity, a disease often fatal to grazing animals

## **Introduction**:
### Population size and standing genetic diversity affect the balance between selection and genetic drift in the evolution of species. 

### The large population sizes of most bacterial species reduce the effects of drift, and selection is predicted to be a more dominant force [(Shapiro et al., 2016)]([[What Is Speciation?]])

### drift has a greater effect on some bacteria, mainly obligate 
endosymbionts and some host-restricted pathogens because bacteria with 
these lifestyles experience population bottlenecks upon each 
transmission and infection

### Class 1, type I CRISPR systems are deﬁned by the cas3 gene that encodes a helicase, often fused to an endonuclease domain
#### depends upon the presence of a protospacer-adjacent motif (PAM) that is essential for acquisition and interference that is proximal to the protospacer

### [[Rathayibacter]] require plant parasitic nematodes of the Anguinidae taxon to be vectored to plants.
#### Juvenile nematodes enter ovules and can induce seed galls. The nematodes can subsequently be displaced by [[Rathayibacter]], which then proliferate and produce gummosis, a slime that is characteristic of the gumming diseases they cause.

### Annual ryegrass (Lolium rigidum) was deliberately introduced as a pasture plant into Australia in 1880. 

### Historical records indicate that annual ryegrass toxicity was ﬁrst reported in South Australia in 1956 and in Western Australia in 1968

### The disease has been reported in more than 10 million hectares of farmland. 

### Animals that consume infected grasses suffer episodic neurological symptoms, often leading to mortality. 

### In Western Australia alone, it has been estimated that from 1968 to 2000, more than half a million sheep died from annual ryegrass toxicity. 

### Toxicity is due to corynetoxin, which interferes with the early steps of protein glycosylation

### Bacteriophage NCPPB3778 is another partner in the ecology of [[Rathayibacter]] toxicus
#### NCPPB3778 is suggested to adopt a pseudolysogenic state, an extended, arrested, and nonreplicative developmental state

### quantiﬁed gene presence/absence polymorphisms and single nucleotide 
polymorphisms and demonstrate that relative to other species of 
[[Rathayibacter]], R. toxicus experienced a process of genome reduction and 
has exceptionally low genetic variation

### 1st phase: ancient and repeated infections by bacteriophage NCPPB3778 triggered population bottlenecks, large-scale gene loss, and emergence of a lineage that acquired the CRISPR locus

### 2nd phase" periodic selection is occurring; NCPPB3778 pressures repeatedly acting on the CRISPR locus cause recurring genome-wide sweeps that reducepopulation genetic diversity.

## **Results**:
### **The Rathayibacter genus consists of at least nine species**:
#### The assembled genomes ranged from 2.3 to 4.4 Mb in size, and the [[G+C content]] ranged from 61.5% to 72.7%

#### intraclade diversity is low, with most pairwise ANI values exceeding 99%. Interclade comparisons ranged in ANI values from 72% to 90%, with the lowest values obtained in pairwise comparisons to R. toxicus.

### **R. toxicus evolved via genome reduction**:
#### The long branch leading to R. toxicus and the high interclade divergence
 in ANI are consistent with an accelerated rate of evolution in this 
species relative to others in the genus

#### ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FUP_SHmCMuv.png?alt=media&token=101ed59d-c6ae-4aab-a70b-246d381b4496)
**Figure S1: Rathayibacter circumscribes nine species.**  Average nucleotide identity (ANI) was used to analyze all possible pairwise comparisons of 
genome sequences (see [Data Set S1B](https://mbio.asm.org/content/9/4/e01280-18/figures-only#DC1) for values). Species-level groups were defined based on an ANI threshold of 94% ANI cutoff.

#### Despite the signiﬁcant decrease in relative size, the R. toxicus genomes of isolates FH232 and FH79 belonging to clades C and A, respectively, have few large-scale rearrangements relative to each other (21) (Fig. 2A).

#### We used a [[Welch’s t test]] to identify 345 orthologous clusters with genes that are signiﬁcantly enriched in presence in the genomes of Rathayibacter relative to those of closely related genera

#### Decreases in [[G+C content]] are often associated with loss of DNA repair pathways. R. toxicus, like all members of [[Rathayibacter]] and many [[Actinobacteria]], lacks homologs of mutS, necessary for mismatch repair

#### Individual isolates of R. toxicus had few unique gene gains subsequent 
to the emergence of the species. R. toxicus is predicted to have the 
smallest average number of [[orphan genes]]

#### GI_SVM, which identiﬁes regions distinct on the basis of compositional bias of [[k-mer]], to determine the contribution of HGT

#### were predicted to differ in compositional bias of k-mers. However, all regions are homologous to at least one region in another genome sequence that GI_SVM did not identify

### **R. toxicus forms genetically homogenous groups**:
#### To test whether low genetic diversity is unique to R. toxicus, we used Rtr2 (clade A) as a comparator group. This group was selected because it was sufficiently sampled for our analyses.

#### Rtr2(clade A) is substantially more heterogenous than the clades of R. toxicus. In the SNP tree, Rtr2(clade A) is partially resolved with some [[polytomies]]

#### A total of 920 single-copy orthologous genes present in all members of R. toxicus and Rtr2 (clade A) were characterized

#### The distribution of nucleotide diversity value per gene is heavily skewed toward 0 in R. toxicus

#### We conﬁrmed, by subsampling 100 times, 22, 11, and 7 randomly selected 
isolates of Rtr2 (clade A) to match the sizes of the R. toxicus groups, that differences are not due to variations in sample size

#### ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FHOquUydP4y.png?alt=media&token=decd4d05-4fac-4aca-bbdc-67097e078c06)

#### **Figure 4: Genetic diversity in R. toxicus groups. ** (A) Whisker box plots 
representing the nucleotide diversity (π) for 920 genes with orthologs in all sequenced isolates of R. toxicus (Rto; 22 total isolates, with 11 and 7 isolates in subclades A1 and B, respectively) and clade A of R. tritici-like isolates (Rtr2; 47 isolates). For the subsets of Rtr2 isolates, plots are the totals of 100 resampled isolates. The confidence intervals were 0.012 to 0.0136 (22 isolates), 0.008 to 0.012 (11 isolates), and 0.008 to 0.013 (7 isolates). (B) Diagram showing the number of genes with evidence for positive selection in all possible comparisons between clades A to C (purple circles) of R. toxicus.

#### Of the 920 R. toxicus orthologous genes, 29% have no polymorphisms when 
examined at the level of the species. 

#### In contrast, Rtr2 (clade A) has no ^^monomorphic genes, no matter the sample size examined^^.

#### the nucleotide diversity of these genes was zero in any of the three clades of R. toxicus and nonzero in all of R. toxicus . Of these genes, 60 genes had a ratio of nonsynonymous to synonymous substitutions ($$dN/dS)$$ of $$>1$$, which is often used as evidence as [[positive selection]]

#### However, all 60 genes are apomorphies, as no ﬁxed allele was identiﬁed in more than one pairwise comparison between clades of R. toxicus.

#### These data are not consistent with divergent selection acting on the 60 ﬁxed alleles.

### 

## **Discussion**:
### an ancient lineage susceptible to bacteriophage NCPPB3778 is hypothesized to have suffered repeated and massive losses to its genome. NCPPB3778 adopts a pseudolysogenic state, which could lead to long-term coexistence with increased frequency in lytic cycles and repeated bottlenecks to the host population

### The periodic resetting of diversity to near zero is highly disruptive to the process of HGT. Reduced frequency of gains, coupled to the inherent deletion bias of bacterial genomes, will result in genome attrition and yield a structurally stable genome that, relative to sister species, has few large-scale rearrangements and is richly punctuated by gene deletions

### A high frequency of phage blooms reduces genetic heterogeneity of 
populations, dampening the effects of recombination and deterring HGT.

### despite sharing a very recently derived common ancestor, among the 60 
genes, there are no alleles common to groups A and B, relative to group 
C. This is not consistent with loci being under positive selection and 
providing a ﬁtness beneﬁt.

### Sampling bias is not likely an explanation for the low diversity in R. 
toxicus groups because the geographically separated isolates showed 
similar patterns and isolates were collected over the span of decades from multiple plant host species

### We also demonstrated that the genetic diversity of R. toxicus was 
substantially less than the diversity in R. tritici-like (Rtr2) (clade 
A) isolates that were collected over a signiﬁcantly shorter period of 
time and in a more geographically restricted location

### Pseudolysogeny is also an indicator of a nutrient-poor condition, one that compromises immunity

## 

## Other notes:
### Bowtie2v2.3.3.1 was used to align sequencing reads from isolates of R. toxicus and R. tritici-like to the corresponding reference sequences of R. toxicus FH232 orR. tritici-like RFBD1 (64). 

### Single nucleotide polymorphisms (SNPs) were identified using the haplotype caller of the genome analysis toolkit (GATK)software (65). VCFR was used to filter for high-quality SNPs (66). 

### To be included in the final analysis, the nucleotide positions had to meet the criteria of having a read depth of 25% to 75% of total reads of the corresponding genome and of being present in80% of the genome sequences.

### The nuc.div function of the APE R package was used to calculate nucleotide diversity for each orthologous group within each set (77). Using the same process, sets of 100 bootstrap replications of 22, 11, and 7 randomly selected isolates from clade A of R. tritici-like isolates were analyzed for nucleotide diversity. Boxplots were generated, using the ggplot2 R package (78).
