---
title: Evolution and Ecology of CRISPR
---

## CRISPR arrays form the genetic memory of the prokaryotic adaptive immune system

## composed of repeating sequencing (repeats) that are interspaced by variable sequence (spaces) - native sequences from mobile genetic elements such as viruses

## __Cas__ genes encode the protein machinery that uses the information stored in the CRISPR assay to launch an immune response against mobile genetic elements that carry a cognate sequence

## spacers are thus acquired during the lifetime of an individual clone, and the resultant phage resistance phenotype is inherited

## 

## 1. Mechanism of CRISPR-Cas systems is divided into 3 stages: adaptation, expression, & interference
### 1. Adaptation: novel spacers are integrated into the CRISPR array

### 2. Expression: CRISPR array is transcribed and processed into short crRNA molecules that are loaded into the __Cas__ proteins

### 3. Interference: __Cas__-crRNA ribonucleotide complexes, bind to complementary nucleic acids, followed by degradation of the target molecule

### A. ^^Adaptation^^ always requires at least __Cas1__ and __Cas2__, which are most conserved __cas__ genes
#### recent __in vitro/in vivo__ studies show the hetertetrameric complex with __Cas1__ and __Cas2__ bind to the leader end of the CRISPR array
##### __Cas1__ catalyzes the integration of spacers similar through a mechanism that shares similarities w/ retrovirus integration

### B. During ^^expression^^, the CRISPR array is transribed into pre-crRNA which is cleaved by a __Cas-b-like__ endonuclease (class 1) or by RNAIII/Cpf (class 2) to yield mature or RNA associate with __cas__ proteins

### During ^^interference^^, cr-RNA-cas complexes bind and cleave the complementary nucleic acid or mark it for destruction by cas

## 2. Evolution of CRISPR-cas systems
### __Cas1__ is the most conserved of the __cas__ proteins

### not all __cas1__ genes are CRISPR associated

### two clades of orphan __cas1__ (__cas1-solo__) were identified from predominantly archeal lineages
#### in some cases __cas-1-solo__ genes are found within predefined transposable elements

#### (casposons-genomic islands, flanked by terminal inverted repeats)

### almost all CRISPR-cas systems carry __cas2__ (shared structural similarity with __vap1__ - a toxin)
#### __cas2__/other cas proteins may act as a toxin to induce cell damage

### class1 systems encode for a suite of genes essential for expression, RAMP family (repeat associated mysterious protein) suggesting a series of gene duplications events
#### carry multiple rrms (single runs recognition motif)

### it has been suggested that class 2 effector nuclease may have acted as a stand alone immune mechanism and had been co-opted from multiple genetic element by CRISPR-cas

## 

## How important are CRISPR-cas systems in nature?
### CRISPR loci belonging to different populations of the same species are typically higher regarding spacer control

### CRISPR spacers tend to mature phage genomes from the same (sympatric) rather than (allopatric) environments
#### demonstrates that CRISPR-based antagonism coevolution occurs at the

### Class 1 systems:
#### encode a multisubunit crRNA-cas complexes subdivided into type 1, 2, 4 systems

#### precursor-crRNA is processed by cas-6 type endonuclease

#### type 1 systems (from e.coli K12) encoded for th complex as cascade (5 cas proteins) and a single crRNA consists of spaced sequences flanked by partial repeats that some as conserved handel

### Question: How does CRISPR cas systes distinguish between beneficial and pathogenic dervied mobile genetic elements?

### Upon binding target, cascade recruits cas 3 endonuclease 

### Major differences between type I and type III:
#### Type 3 target both single stranded RNA and trans.....

### CRISPR may preferentualy target rare viruses

### lack of spacers is likely due to low sequence availability

## 

## When do CRISPR-cas systems matter?
### Different strains of the same species often differ with regard to pressure, number, length of CRISPR loci

### Ecological features determine the relative benefit
#### a. virus genetic diversity

#### b. force of infection

#### sequence specific interaction between them increases bacterial and viruses

### If we predict/manipulate the evolution of CRISPR-cas arrarys and associated co-evolution, we need to understand the selective forces that drive its evolution

### Priming refers to the partial match between a pre-existing spacer and the genome of an invading phage

### Note: mechanism of specificty in cascade for determinening vs. interference
