---
title: Identification of innate immunity elicitors using molecular signatures of natural selection
---

- **Authors**: [[Honour C. McCann]], [[Nardeep Nahal]], [[Shalabh Thakur]], and [[David S. Guttman]]
- **Journal**: [[PNAS]]
- **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:b747ceee-109f-43e6-aaa2-53e041b32156
- **Tags**: #[[plant immunity]] #[[MAMPs]]
- **Abstract**:
	- we reasoned that hosts are more likely to evolve recognition to “core” pathogen proteins under strong negative selection for the maintenance of essential cellular functions, whereas repeated exposure to host–defense responses will impose strong positive selective pressure for elicitor  diversiﬁcation to avoid host recognition
- **Introduction**:
	- [[PRR-Triggered Immunity]] elicitors may therefore be characterized as proteins that are encoded by ubiquitous and conserved genes (i.e., components of the core genome) that have discrete regions containing potentially high rates of [[nonsynonymous]] (amino acid changing) substitution
	- under very strong [[negative selection]] as a whole to preserve their 
	  critical function, but which have distinct regions under very strong 
	  [[positive selection]] to avoid host recognition.
- **Results**:
	- **Detection of Positive Selection in the Core Phytopathogen Genome**:
		- The core genome was operationally deﬁned as the [[Orthologous]] gene set 
		  common among the ﬁnished (closed) genomes of [[Pseudomonas syringae]] pv. tomato DC3000 (PtoDC3000), P. syringae pv. phaseolicola 1448A, P. syringae pv. syringae B728A, [[Xanthomonas campestris]] pv. campestris  33913, X. campestris pv. campestris 8004, and X. campestris pv. 
		  vesicatoria 85-10.
		- In all, 1,322 [[Orthologous]] core genes were identiﬁed by using the [[reciprocal smallest distance algorithm ]]
			- which is less vulnerable to ortholog exclusion because of paralog identiﬁcation in one BLAST direction
		- 35% of the core genes were found to have at least one positively selected residue
		- ikelihood ratio test (LRT) rejected the neutral and [[negative selection]] 
		  null model (M7) in favor of the [[positive selection]] model (M8) for 56 
		  proteins [P < 0.01, corresponding to a false discovery rate (FDR) of 
		  10%; Table S1].
		- [[Bayes Empirical Bayes]] (BEB) estimates from model M8 were then used to 
		  identify the putatively positively selected sites for each candidate
		- Flagellin, which is the best characterized [[Pseudomonas syringae]] [[MAMPs]], also had multiple positively selected residues; nevertheless, we could not reject the null hypothesis for this locus at the P = 0.01 level because the intensity of selection on these residues is substantially weaker (ω <  1.5) than that observed for many other candidates including EF-Tu.
		- Bacteria are exposed to a variety of [[selective pressures]] that can give 
		  rise to similar patterns of [[positive selection]] among the core genome. 
		  For example, some of these proteins may serve as bacteriophage receptors, selecting for increased genetic variation to avoid phage predation.
		- additional evolutionary analysis on the 56 candidate elicitors from 
		  seven soil-inhabiting, nonpathogenic Pseudomonas strains (P. ﬂuorescens 
		  strains Pf-5, Pf0-1, SBW25 and P. putida strains F1, GB-1, KT2440, and 
		  W619
			- nonpathogenic soil microbes will not be exposed to the same host imposed 
			  selective pressures as plant-associated pathogenic microbes, suggesting these proteins are less likely to be targets for plant recognition
		- [[elf18]] epitopes derived from several genera of phytopathogenic bacteria and showed that they have variable defense elicitation activities in Arabidopsis.
		- positively selected amino acids will be clustered into islands that correspond to the speciﬁc elicitor epitopes
		- clustered within a 30-aa window, we only considered loci with two or more positively selected residues to control for biases imposed by a general excess of positively selected sites among the candidates.
		- 50% of all candidate elicitors had clusters of positively selected residues, whereas this pattern was found in only 6% of the entire core genome
	- **Functionally Assaying Candidate Elicitors for Immune Elicitation**:
		- signiﬁcant statistical evidence of positive selection under the M7-M8 
		  likelihood ratio test, and [[Bayes Empirical Bayes]] estimates identifying one or more positively selected residues
		- candidate elicitor epitopes based on the location of the positively 
		  selected sites within these proteins and synthesized three classes of 
		  peptides ranging in size from 20 to 40 aa
		- The virulence suppression activity of each candidate was assayed by 
		  preinoculating A. thaliana ecotype Col-0 with the peptides followed by 
		  an inﬁltration 24 h later with either PtoDC3000
		- signiﬁcantly reduced bacterial growth compared with the water control, although not to the same degree as ﬂg22Pto.
		- negative candidate peptide PSPTO5086-3:b showed no effect on bacterial growth compared with a mock inoculation
		- virulence suppression results with this peptide were highly inconsistent across assays and more typically did not induce a signiﬁcant immune 
		  response
		- pooling three negative peptides had no greater effect than the individually tested peptides and did not signiﬁcantly decrease
		- signiﬁcantly suppressed virulence, but the effect was not signiﬁcantly 
		  greater than that observed for one of the three positive candidates 
		  tested individually
		- 8 of 10 positive candidate peptides showed signiﬁcant and consistent virulence suppression actvity
		- the only class a candidate peptide that did not show any virulence 
		  suppression activity was also recovered as a marginally signiﬁcant 
		  candidate (PSPTO2757-2:a; P = 0.02) in the analysis of the nonpathogenic
		   Pseudomonads.
		- We elected to test the candidate elicitors ability to induce the 
		  formation of cell wall papillae via the well-established callose 
		  formation assay
		- BSA as a nonspeciﬁc protein control, and water as a negative control. 
		  Peptides were inﬁltrated into A. thaliana, and leaves were stained with 
		  aniline blue 24 h after inﬁltration. Callose deposition was monitored 
		  via epiﬂuorescent microscopy and analyzed with ImageJ
		- found that eight of nine positive candidates peptides resulted in 
		  signiﬁcantly greater callose than the water control by using an FDR of 
		  0.05%
- **Discussion**:
	- One possible explanation for the weaker overall pattern of signiﬁcance 
	  relative to the virulence suppression assay is that this assay measures a
	   speciﬁc cellular reaction as opposed to nonspeciﬁc immune response, and
	   we do not know the potential spectrum of cell wall-associated responses
	   induced by MTI elicitors.
	- There is clear evidence of natural genetic variation in both MAMPs (9, 
	  13, 14) and their cognate PRRs (22) affecting defense induction and the 
	  effectiveness of the immune response
	- Nevertheless, as Boller and Felix (8) point out, many well-deﬁned MAMPs 
	  have unknown receptors. Examples include bacterial cold shock protein, 
	  superoxide dismutase, harpins, a secreted transglutaminase, strerol-binding elicitins, and a cellulose-binding lectin from oomycetes
	- It is also extremely difficult to understand why proteins would carry the specific selective signatures used in this screen if they did not interact with specific host receptors
	- This study has illustrated how patterns of natural selection in 
	  bacterial proteins can be used for identifying inducers of innate 
	  immunity. T
- Other Notes
	- Statistical test used: [[Fisher’s exact test]]