---
title: Multiple plasmid-borne virulence genes of Clavibacter michiganensis ssp. capsici critical for disease development in pepper
---

- **Authors**: [[In Sun Hwang]], [[Eom-Ji Oh]], [[Donghyuk Kim]], and [[Chang-Sik Oh]]
- **Journal**: [[New Phytologist]]
- **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:a50a17fd-57dd-4c97-a7c7-a28359893ea4
- **Tags**: #[[Comparative Genomics]] #clavibacter #[[Clavibacter capsici]]
- **Abstract**:
	- [[Clavibacter capsici]] type strain [[PF008]] carries one chromosome (3.056 Mb) and two plasmids (39 kb pCM1Cmc and 145 kb pCM2Cmc)
	- bacterium lacks a chromosomal pathogenicity island and celA gene, but carries most putative virulence genes in both plasmid
	- [[chpE]], [[chpG]], [[ppaA]]1, [[ppaB1]] and [[pelA1]], encoding [[serine proteases]] or pectate lyase contribute to disease development in pepper.
- **Introduction**:
	- [[Clavibacter capsici]], causing bacterial canker in pepper, was recently classified, although bacterial strains have been isolated from red peppers and bell peppers in Korea and the USA, respectively, since the 1990 (Lewis & Miller, 2000; Yim et al., 2012)
	- pathogenicity island [[chp/tomA]], which include multiple virulence genes mostly encoding enzymes such as pectate lyase, [[serine proteases]], and tomatinase, and this PAI region has been shown to play an important role in invasion of the host plant
	- although some genes in the PAI region of [[Clavibacter michiganensis]] are present in other chromosomal regions and plasmids
	- [[Clavibacter sepedonicus]] strain ATCC33113 carries two plasmids, [[pCS1]] (50 kb) and pCSL1 (95 kb) (Bentley et al., 2008), and its two known virulence genes, celA and chp-7 gene homologous to pat-1 gene of [[Clavibacter michiganensis]], are located in [[pCS1]] and a chromosome, respectively (Nissinen et al., 2009)
	- [[Clavibacter capsici]] strain PF008 carries two plasmids, pCM1Cmc (39-kb) and pCM2Cmc (145-kb) (Bae et al., 2015).
	- whole genome sequences of nonpathogenic strains, such as [[Clavibacter sp.]] [[CF11]] isolated from soil at a tomato-planting glasshouse and [[Clavibacter sp.]] LMG26808, have also been reported
	- Elevated expression of genes encoding other plant [[CAZymes]], such as pelA encoding a pectate lyase, xysA and xysB encoding xylanases, and/or the chp genes encoding [[serine proteases]], were associated with disease development of [[Clavibacter sp.]] (Holtsmark et al., 2008; Chalupowicz et al., 2010)
	- Moreover, [[vatr1]] and [[vatr2]] encoding transcriptional regulators of [[Clavibacter michiganensis]] were recently shown to be implicated in
	   a virulent interaction with host plants via regulation of ethylene production (Savidor et al., 2015)
	- In [[Clavibacter sepedonicus]], putative virulence genes, including [[celA]] and [[celB]] genes, were up-regulated during potato infection (Burger et al., 2005; Holtsmark et al., 2008).
- **Results**:
	- **Genome profiles of the C. michiganensis ssp. capsici strain PF008**:
		- [[NCPPB  382]], which has a larger chromosome (3297 kb) and smaller plasmids of pCM1 (27 kb) and pCM2 (70 kb)
		- blastn analysis showed that pCM1Cmc and pCM2Cmc of [[Clavibacter capsici]] strain PF008 have query coverages of 53% and 25% respectively
		- In addition, both plasmids of strain [[PF008]] and  strain [[NCPPB  382]] have the same plasmid backbone, including replication protein or partitioning protein, indicating that these plasmids might be classified into the same family
		- PF008 lacks [[celA]], which is a key virulence gene of [[Clavibacter michiganensis]], while the [[celB]] gene was found on the chromosome, as in [[NCPPB  382]] (Fig. S1c)
	- **Exploring the virulence regions of C. michiganensis ssp. capsici strain PF008**:
		- pgaA, ppaH, and celB, encoding polygalacturonase, extracellular serine protease, and cellulase, respectively, were also found on the chromosomes of [[PF008]]
		- homologs to some genes in the chp subregion of [[Clavibacter michiganensis]] were found in either pCM1Cmc or pCM2Cmc
		- Over 90% DNA sequence identity between those two virulence regions in [[PF008]] suggests that these virulence regions are duplicated and might be important for disease development in host plants.
		- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FC_F5waPxVb.png?alt=media&token=67f4fe99-caa6-49df-8f0a-0923495fd38b)
		- [[PF008]] had three pelA paralogs, one (pelA1) in the pCM1Cmc and two (pelA2 and pelA3) in pCM2Cmc plasmids, which are orthologous to two pectate lyase genes present in [[chp/tomA]]
		- tomA subregion was completely missing [[PF008]]
		- [[vatr1]] and [[vatr2]], encoding virulence-associated transcriptional regulators, are also located near celB in the chromosome of PF008 (Fig. 1a; Table 1)
		- Overall, among the 28 putative virulence genes listed in Table 1, 11, 
		  including celA, were missing in PF008, and this might be related to the lack of wilting symptoms caused by [[Clavibacter capsici]] in pepper compared with those caused by [[Clavibacter michiganensis]] in tomato
	- **Pathogenicity test of C. michiganensis ssp. capsici strain PF008 in pepper plants**:
		- we could only successfully generate a pCM1Cmc deletion mutant (PF008ΔpCM1Cmc) by this method.
		- three PF008DpCM1Cmc strains induced disease symptoms in the pepper plants, but these symptoms were delayed or mild compared with those  produced by the wild-type strain (Fig. 2b,c)
		- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FITiMC43Lgj.png?alt=media&token=5e33f1f1-6a13-4524-a542-5daead43272a)
		- whether plasmid deletion can affect bacterial survival in vivo and/or in vitro, we tested bacterial growth of the wild-type and PF008ΔpCM1Cmc strains in liquid media and in pepper plants
		- bacterial growth of the PF008ΔpCM1Cmc strain on KB broth medium was significantly delayed (Fig. 3a)
			- maybe some genes in pCM1Cmc are involved in basic bacterial metabolism.
		- as most of the genes on pCM1 Cmc were annotated as genes encoding hypothetical proteins, this remains speculative.
		- bacterial cell numbers of PF008ΔpCM1Cmc strains were slightly, but 
		  significantly, reduced compared with that of the wild-type strain in plants only at 3 dai
		- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FXucIkFh9XP.png?alt=media&token=a9fc4bd5-3c8a-4656-b9d9-4460dc686a22)
		- deletion of plasmid pCM1Cmc did not induce suppression of its colonization during infection at 4 dai, although PF008ΔpCM1Cmc strain-infected plants exhibited significantly reduced and delayed disease symptoms compared with 
		  wild-type-infected plants (Fig. S2)
		-
	- **Expression profiles of the virulence genes of C. capsici wild-type and PF008ΔpCM1Cmc strains in pepper plants during infection**:
		- To determine the difference in expression of virulence genes between wild-type and PF008ΔpCM1Cmc strains during host infection, we analyzed expression of putative virulence genes by qRT-PCR.
		- Expression of three genes, ppaA, ppaB and pelA, which are located in both plasmids, was affected by deletion of pCM1Cmc during infection, even though pCM2Cmc has identical twins of these genes
		- Despite the severe necrotic symptoms of pepper leaves infected with the wild-type strain, the expression of celB was suppressed in leaves infected with both wild-type and PF008ΔpCM1Cmc strains (Fig. 4d), indicating that celB expression is not correlated with development of severe disease symptoms in infected leaves.
		- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FoQFrQTP76Z.png?alt=media&token=77ab21eb-faad-47b2-8bb2-f3279328edfe)
	- **Complementation of C. michiganensis ssp. capsici PF008DpCM1Cmc strain with putative virulence genes in pCM1Cmc**:
		- Pepper plants inoculated with
			- PF008ΔpCM1Cmc(chpE-ox)
			  PF008ΔpCM1Cmc(ppaA1-ox)
			  PF008ΔpCM1Cmc(ppaB1-ox) 
			  PF008ΔpCM1Cmc(pelA1-ox)
			  PF008ΔpCM1Cmc(chpG-ox)
		- displayed rapid and strong necrotic symptoms compared with plants inoculated with the PF008ΔpCM1Cmc strain.
		- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FNnRWGqzqA6.png?alt=media&token=6c8ae927-b6c1-44f9-9d6d-8a9042821843)
- **Discussion**:
	- these different repertoires of putative virulence genes in [[clavibacter]] might be tightly correlated with disease severity in host plants; in particular, less putative virulence genes in [[Clavibacter capsici]] might be correlated with mid disease symptoms in pepper.
	- There are two possible reasons for the different locations of virulence genes between [[Clavibacter capsici]] and [[Clavibacter michiganensis]].
		- 1) some regions in the chromosomal [[chp/tomA]] PAI were relocated into pCM1Cmc or pCM2Cmc of [[Clavibacter capsici]] through uncharacterized recombination processes, during which virulence regions were duplicated in both plasmids.
		  2) [[Clavibacter capsici]] might be more similar to an ancestor strain, and most of the virulence genes in the two plasmids of [[Clavibacter capsici]] are relocated to the chromosome of [[Clavibacter michiganensis]]
	- pelA gene of pathogenic bacteria, a pectate lyase, is known to degrade 
	  plant cells and promote bacterial growth using the degraded pectin as a 
	  carbon source
	- nonpathogenic Clavibacter strains, such as C. michiganensis strain LMG26808, lack these genes
	- These expression patterns were somewhat unusual, but can be explained by two possibilities.
		- 1) the presence of putative transcriptional regulator(s) in pCM1Cmc, which might control expression of these virulence genes.
			- In pCM1Cmc, there is one gene (PF008 locus tag: AES38_RS14530) that is highly homologous to the transcriptional regulator arsR of Frigoribacterium sp. and Curtobacterium sp., which can function as a transcriptional repressor of an arsenic resistance operon and regulates an operon implicated in biofilm growth of plant-associated bacteria (Barbosa & Benedetti, 2007).
		- 2) The other is lower copy numbers of these genes in PF008ΔpCM1Cmc as a 
		  result of deletion of pCM1Cmc, compared with the wild-type strain.
	- the expression of the plasmid-borne genes, [[celA]] and pat-1, of [[Clavibacter michiganensis]] is affected by the presence of the [[chp/tomA]] PAI region during infection of a host plant, and the deletion of plasmid(s) affects the expression of chromosomal virulence genes in PAI and/or [[celB]].
	- The virulence mechanism contributing to the pathogenicity in [[PF008]] during infection seems to be more dependent on the activity of the serine protease and/ or pectate lyase than on cellulase for invading a host plant
	- Similarly, [[Clavibacter nebraskensis]] and [[Clavibacter insidiosus]] also carry only [[celB]], but lack a celA gene (Zaluga et al., 2014)
	- suggesting that virulence genes and/or factors located in plasmids contribute to the virulence of [[Clavibacter capsici]] during its interaction with pepper plants.
	- Our results also suggest that the pathogenicity and host specificity of the [[clavibacter]] species depend on a virulence mechanism that has evolved to adapt to certain host plants; thus, [[Clavibacter capsici]] might have evolved alternative strategies to survive, and the differences in pathogenicity between  [[clavibacter]] species might have occurred through several recombination events during separate evolutionary paths in different environments or host plants.
- Other Notes
	- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FzHl_h5C3qy.png?alt=media&token=f21f8219-18de-468c-8538-6f86a7777111)
	- ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FXx5oGPpc5N.png?alt=media&token=b331e28a-0028-43bc-b31d-3430f7fa9fc3)
	  Fig. S1 Genome profiles of PF008. (a) Comparison  of the 200-kb chromosomal pathogenicity island (PAI) region of Cm ssp. michiganensis strain NCPPB382 with other subspecies. Sequence-matched regions are colored, and missing regions are blank  in the PAI box. This analysis was performed using the Genome Evolution tool GEvo from the [CoGe comparative genomics package](http://genomevolution.org/CoGe/GEvo.pl). (b) A comparison of the PAI region using a draft genome sequence from Cm strain CF11. The PAI region of Cm ssp.michiganensis strain NCPPB382, which is approximately from 40 Kb to 170 kb on its genome, and the first 200 kb 
	  of Cm ssp. capsici strain PF008 was compared to that of Cm strain CF11. 
	  Grey and black lines  indicate sequence-similar  regions  analyzed by matching  the strain CF11-contigs to the assigned regions of Cm ssp. michiganensis strain NCPPB 382 and Cm ssp.capsici strain PF008,  respectively. (c) Comparison of plasmid numbers and important virulence  
	  factors among C. michiganensis strains. Yes, present; no, not present; 
	  pseudo, present but not functional;  ND, not determined.  NCPPB382, 
	  Cmssp. michiganensis; NCPPB2581, Cm ssp. nebraskensis; ATCC33113, Cmssp. sepedonicus; R1-1, Cmssp. insidiosus; LMG 26808 and CF11, non-pathogenic Cm.