---
title: Phylogenetic tree building in the genomic age
---

## **Authors**: [[Paschalia Kapli]], [[Ziheng Yang]], and [[Maximilian J. Telford]]

## **Journal**: [[Nature Genetic Reviews]]

## **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:17c1e0ae-5faa-47b9-8003-ae057de9cd8a

## **Tags**: #phylogenetics #genomics

## **Abstract**:
### Understanding the different sources of errors and the strategies to 
mitigate them is essential for assembling an accurate tree of life

## **Introduction**:
### [[Homologous]]: When features, including morphological characters and gene 
loci, are inherited from a common ancestor, for example, a gene in two 
species originating from a single ancestral gene.

### [[Orthologous]]: Homologous sequences that have diverged due to speciation events.

### More generally, character mapping allows us to follow the changing 
character states across a tree to reconstruct the historical path of 
evolution.

### we outline the various methodological choices and several corresponding 
trade-offs between model sophistication and computational demands.

### We then discuss how to align orthologues from different species to 
account for insertions and deletions, and strategies for trimming 
unreliably aligned regions. 

### Finally, we discuss in detail the choice of inference methods and substitution models and consider potential errors as well as approaches to identify and avoid or mitigate them.

## **Orthology predictions.**:
### Two genes are homologous if they are inherited from an ancestral gene 

## 

## 

## **Discussion**:
