---
title: The nature and identification of quantitative trait loci: a community’s view
---

- **Authors**: Members of the Complex Trait Consortium
- **Journal**: [[Nature Genetic Reviews]]
- **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:b1a140e5-dd32-4152-be3d-60e631d59c9b
- **Tags**:
- **Notes**:
	- Much of the genetic variation that underlies disease susceptibility and 
	  morphology is complex and is governed by loci that have quantitative 
	  effects on the phenotype
	- Gene–gene and gene–environment interactions are common and make these loci difficult to analyse
	- **Definition of a QTL**
		- A quantitative trait is one that has measurable phenotypic variation owing to genetic and/or environmental influences.
		- This variation can consist of discrete values or can be continuous
		- A [[QTL]] is a genetic locus, the alleles of which affect this variation
		- Generally, quantitative traits are multifactorial and are influenced by several polymorphic genes and environmental conditions, so one or many [[QTL]]s can influence a trait or a phenotype
		- phenotypic variation can also be caused by environmental factors that 
		  are independent of genotype or through gene–environment interactions
		- These are difficult to separate by recombination events and therefore might be detected as one [[QTL]]
		- Two classic examples of quantitative traits are height and weight
		- These traits can also be influenced by loci that have large discrete effects (often called MENDELIAN LOCI)
		- example, genes that cause dwarfism also affect height but in a qualitative ‘all-or-none’ way.
		- MODIFIER LOCI that modulate the effects of a Mendelian locus can also be described as QTLs.
		- The distinction between Mendelian loci and [[QTL]]s is artificial, as the same mapping techniques can be applied to both.
		- one end of the spectrum is the dichotomous Mendelian trait with only two detectable and distinct phenotypes, which are governed by a single gene
		- the other end are traits, such as growth, which are likely to be affected by many genes that each contribute a small portion to the overall phenotype.
		- Generally, the more loci that are involved in determining a quantitative trait, the more difficult it is to map and identify all of the causative [[QTL]]s
		- each might have a different effect size and the effects of individual [[QTL]]s will vary from strong to weak
	- **Coarse mapping**
		- Coarse mapping is mapping to a chromosomal segment, usually within a range of 10–30 centimorgans (cM)
		- likelihood of success in [[QTL]] mapping depends on the heritability of the trait, its genetic nature (dominant, recessive or additive) and the number of genes that affect it
		- the mapping resolution depends on the number of recombination events in the mapping
		- Mapping [[QTL]]s with smaller effect sizes requires larger mapping populations.
		- [[QTL]] interval or the critical region (defined as the region that must 
		  contain the candidate locus on the basis of recombination breakpoints on
		   either side of the candidate locus)
	- **Fine mapping**
		- Fine mapping (to <1–5 cM) is difficult as it requires more recombination events to separate the genes that govern the quantitative trait from closely linked markers
		- Crosses that involve many recombination events are the most successful
		- If other types of cross are used to narrow the critical region, PROGENY TESTING is often necessary to confirm the phenotype of the recombinant 
		  mice.
		- The size of the critical region that is required for successful results varies depending on whether the region is gene-rich or gene-poor
	- **Identification of candidate genes**
		- states that the most conclusive evidence for a [[QTL]] resides in the ability to replace one allele with another and test for function, for example, by making a knock-in mouse.
		- Below, we list some of the methods that can be used to identify genes that determine a quantitative trait.
		- __Polymorphisms in coding or regulatory regions.__ Sequence differences that lead to changes in either the structure or regulation (or quantity) of a gene product should be detected between the strains that are used for mapping and are known to differ in the quantitative trait.
		- __Gene function.__ As well as affecting either the structure or regulation of a gene product, some evidence should support a link between the function of the gene and the expression of the quantitative trait being analysed, either by involvement in an appropriate pathway and/or by expression in the appropriate target tissue(s) or cell type(s).
		- __Transgenesis__. Transgenesis with bacterial artificial chromosomes (BACs) or other large chromosomal segments can also be used to confirm the identity of the candidate gene
			- BAC libraries must be available that contain the appropriate alternative alleles at the relevant QTL
		- __Knock-ins.__ Knock-ins can also be used to confirm candidate genes, as replacement of one allele with another at the candidate [[QTL]] should alter the quantitative trait.
		- __Deficiency-complementation test.__ If a knockout (or a null allele) of the candidate QTL is available, then complementation tests between the knockout (or mutant strain) and the strain that contains the QTL variant allele could be used as evidence of gene identity
		- __Mutational analysis.__ With the advent of more convenient mutational techniques, such as the chemical mutagenesis of embryonic stem cells, it is now possible to carry out gene-specific mutational analyses; that is, to collect an allelic series of mutations in a specific gene