---
title: The population genetic test Tajima’s D identifies genes encoding pathogen-associated molecular patterns and other virulence related genes in Ralstonia solanacearum
---

## **Authors**: [[Noam Eckshtain-Levi]], [[Alexandra J. Weisberg]], and [[Boris A. Vinatzer]]

## **Journal**: [[Molecular Plant Pathology]]

## **Readcube**: https://www.readcube.com/library/95fd6ff6-fca1-4a81-9ddb-c57079c1e1f2:cf4b548e-49b9-48e5-abc6-710df11f281c

## **Tags**: #[[PRR-Triggered Immunity]] #[[MAMPs]] #[[Ralstonia solanacearum]]

## **Abstract**:
### Until recently, elf18, an epitope of elongation factor-Tu (EF-Tu), was the sole confirmed PAMP of Ralstonia solanacearum, the causal agent of bacterial wilt disease, limiting our understanding of R. solanacearum–plant interactions

### hypothesis that genes encoding PAMPs are under selection to avoid recognition by plant PRRs

### The screen flagged 49 non-neutrally evolving genes, including not only EF-Tu but also the gene for Cold Shock Protein C, which encodes the PAMP [[csp22]].

## **Introduction**:
### Common characteristics of PAMPs are high abundance, high sequence  conservation and necessity for microbial survival (Zipfel, 2008)

### immune response can be overcome by pathogens through several strategies, mainly via the delivery of PTI-suppressing effector proteins (Macho and Zipfel, 2015) and mutations in the PAMPs themselves leading to reduced, or even absence of, recognition (Rossez et al., 2015; Vinatzer et al., 2014**Results**:

### Hence, there is great interest in finding new approaches to control diseases caused by this important pathogen. The ectopic expression of [[PRRs]] in crops susceptible to R. solanacearum has been shown to be one promising strategy. For example, the expression of the Arabidopsis thaliana EFR receptor of elf18 in potato and tomato has been shown to reduce susceptibility to R. solanacearum (Boschi et al., 2017; Lacombe et al., 2010).

### We decided to identify core genome genes that do not evolve neutrally by calculating Tajima’s D, which is a measure of deviation from neutral evolution (Tajima, 1989)

### Genes with a [[Tajima's D]] value below -2 have an excess of rare alleles, indicating positive selection or a selective sweep.

### genes with a [[Tajima's D]] value above +2 have an excess of common alleles suggestive of balancing selection

### [[Pseudomonas syringae]] fliC gene, which encodes the two flagellin PAMPs flg22 and flgII-28 (Cai et al., 2011), has a Tajima’s D value of 2.269 in a group of Pseudomonas syringae pv. tomato relatives (Monteil et al., 2013)

### 

## **Results**:
### ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FKwUccy8Vzu.png?alt=media&token=5c525653-2f76-4c26-a2d8-057d463b96d9)

### The Tajima’s D value for fliC was 21.53, suggesting that flagellin evolves neutrally in [[Ralstonia solanacearum]]. Although counterintuitive at first sight, this is in agreement with results from previous studies which showed that the R. solanacearum flagellin epitope flg22 is not recognized by any tested plant host (Pfund et al., 2004)

### ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2F_vjsIHjWtN.png?alt=media&token=065b23bb-d53f-43e4-8a0a-3cf7aec8ad0d)

### An R. solanacearum [[csp22]] peptide based on the genome sequence of R. 
solanacearum strain POPS2, corresponding to allele 5, was synthesized.

### ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2FUjIb5UdsRa.png?alt=media&token=459b6938-e775-415d-b2dc-dbc5660d0baf)

**Peptides corresponding to both the [[csp22]] consensus and [[csp22]] of Ralstonia solanacearum allele 5 elicit the production of reactive oxygen species (ROS) in leaves of 4‐week‐old tomato** (A) and 6‐week‐old Nicotiana benthamiana (B), whereas Xup25_Rs is not recognized, using a luminol‐based ROS detection assay. All peptides were tested at a concentration of 1 μm. n = 8 for all plant–peptide combinations in all experiments. Error bars represent standard errors. Similar results were obtained in three independent experiments.

### one of the tagged genes, acrB (the seventh most negative in our list), is part of the acriflavine resistance multidrug efflux system together with acrA, located as 17th most negative in our list.

## **Discussion**:
### fliC alleles are derived from an ancestral allele that had already evolved to escape recognition

### The AcrAB-TolC efflux pump is able to transport a diverse array of compounds with little chemical similarity, thus conferring resistance to a broad spectrum of antibiotics (Anes et al., 2015

### acrA has already been shown to contribute to virulence in R. solanacearum (Brown et al., 2007). Homologues of acrA and/or acrB in P. syringae (Fan et al., 2011; Stoitsova et al., 2008; Vargas et al., 2011, 2013) and in Erwinia amylovora (AlKarablieh et al., 2009; Burse et al., 2004) have also been shown to play important roles in plant–microbe interactions

### not every gene flagged by [[Tajima's D]] is necessarily a gene under selection, as Tajima’s D also identifies genes that have unusual allelic variation because of demographic history or genetic linkage to genes under selection

## Other notes:
### When the obtained P values were corrected to account for the large number of performed tests using a [[Benjamini–Hochberg correction]] (BH), they increased to 0.8 and above, which reveals the possibility that some of the genes identified to not evolve neutrally based on their [[Tajima's D]] value are false positives

### Leaves of four hosts were used: tomato (Solanum lycopersicum) cv. ‘Rio Grande’, A. thaliana ecotype ‘Columbia’, Nicotiana benthamiana and long Italian purple aubergine (Solanum melongena)
