---
title: USDA NIFA Predoctural Fellowship - Project Narrative - Extra Text
---

- **Aim
   I. Investigate the diversity and evolution of pathogenic Clavibacter 
  species and their effectors implicated in host range and pathogenicity. **
- Bacterial
   pathogens secrete protein effectors to evade host immunity, enable 
  effective colonization, and cause disease. For Clavibacter, effectors 
  are secreted to their surrounding extracellular space via the Sec and 
  Tat protein secretion systems (Savidor et al., 2011; Peritore-Galve et al., 2019). Effectors which provide a fitness advantage for the bacterium in a given
   environment are selected for and maintained within the genome. 
  Comparative genomics has become a robust method for in silico prediction
   and analysis of genes and gene families important in biological 
  processes (Loper et al., 2012; Melnyk et al., 2019; Levy et al., 2018; Beskrovnaya
   et al., 2020). I hypothesize that a comparative genomics approach will 
  enable the discovery of loci which contribute to host range and 
  pathogenicity in Clavibacter species. I will generate ~60 high quality 
  genomes for pathogenic Clavibacter not well sampled in NCBI (Fig. 4A). 
  With over 140 genomes, I will investigate the overall diversity and 
  evolution of the genus, predict effectors, and characterize 
  contributions of effectors toward pathogenicity and host range.
- **Aim II. Characterize the interactions between Clavibacter michiganensis and plant immunity. **
- Biotic colonizers possess conserved microbial elicitors which can be perceived by surface-localized plant immune receptors (Boutrot
   and Zipfel, 2014). These microbial associated molecular patterns 
  (MAMPs) can be found in diverse pathogens including Gram-negative 
  bacteria, insects, fungi, and nematodes (Choi and Klessig, 2016). Upon 
  perception, an immune response is elicited as a mechanism to limit 
  pathogen attack. However, the importance of immune perception in G+ 
  pathogens and whether these bacteria carry specific MAMPs remains 
  unknown (Eichenlaub and Gartemann, 2011; Thapa et al., 2019). I hypothesize that G+ plant pathogens carry distinct MAMPs
   that can elicit an immune response in plant genotypes carrying cognate 
  receptors. In addition to testing the response for well-characterized 
  MAMPs encoded in the Clavibacter genome, I will query for G+ MAMPs 
  conserved across the Clavibacter genus and test for perception and immune response on diverse plant species.