---
title: You Lu
---

## Email correspondence with [[Carol A. Ishimaru]] on [[December 7th, 2020]]
### I checked with You Lu about a summary table for genome sizes. He hadn't made one, but reminded me that all the information is in the .gbk annotation files.  I opened the files with a text editor and realized the strain names might be confusing.  It's because Ryan McNally, who submitted the DNA for sequencing, gave aliases to the strains.  In one of the documents I sent, You Lu verified the real strain IDs with the PacBio pseudonyms. Sorry for the confusion.  Here they are for your convenience.

### A01 = CIC292

### B01 = CIC313

### C01 = CIC326

### D01 = CIC328 (not clavibacter so not included in genome comparison or annotated)

### E01 = CIC395

### F01 = CIC396

### The genbank sequence of the type strain NCPPB2581 (=CIC402 in our collection) was used as a reference.

## Email correspondence from [[September 5th, 2019]]
### I have manually curated the genome annotations and examined the sequences underlying some genome content differences. Below is a summary of what I have found so far.

### 1) All the six genomes analyzed (CIC292, CIC313, CIC326, CIC395, CIC396 and NCPPB2581) show high degree of collinearity, and no obvious pathogenicity island was identified. There are small insertions (ranging from 1 to 14 kb, may be of phage origin) in some genomic regions. Some insertions are unique to one isolates, while some are present in several isolates. Interestingly, quite a few of those insertions are identical in CIC313 and NCPPB2581, making the genome of CIC313 looks more similar to the reference in this regard than to the other ones.

### 2) As mentioned in the previous email, I compiled a list of nineteen genomic loci differences of interest, whose underlying nucleotide sequence changes are due to 1-nt deletion within a long string of C or G. I highly recommend verifying these "mutations" by PCR and Sanger sequencing. For each locus, the sequence from one or more isolates needs to be checked. so the total number of PCR reactions and Sanger sequencing will be approximately 50. The gene mutations I discuss below are the ones that can be called out with high confidence (e.g. single nucleotide substitution, or small deletion occurred outside of repetitive sequences, etc).

### 3) The highly virulent CIC395 isolate does NOT encode additional genes relative other isolates, but some gene mutations in the other isolates may have lessened their virulence, rendering them less virulence or avirulent. For example, relative to CIC395, the less virulent clone CIC396 accumulated two mutations that cause frameshifts in carB gene (Carbamoyl-phosphate synthase large chain) and in another gene encoding a drug/H+ efflux antiporter of major facilitator superfamily. In CIC313, a deletion of 139 nucleotides truncated a secreted pectinesterase, which might be involved in cell wall degradation, a nonsense mutation truncated a permease component of a putative ABC-type sugar transporter, and a deletion of 10-nt knocked out rnc gene, which encodes Ribonuclease III. In CIC326, a gene encoding a ribonuclease was frameshifted. These are just several examples, and there are quite a few mutations in other genes, but need verification of the underlying nucleotide sequences as described in 2).

### 4) Some pseudogenes/mutations only present in virulent isolate, while remaining as intact genes in avirulent isolates, may have arisen from negative selections. In other words, presence of the protein/enzyme or its products may elicit immune responses in the host, and the pathogen could benefit from getting rid of the protein/enzyme hence reducing the elicitors that the host can perceive. For example, the plcN gene encoding a nonhemolytic phospholipase C enzyme is uniquely present in CIC313, which might be involved in lipid metabolism. Along with this line, some pseudogenes in C. nebraskensis have functional homologs in other Clavibacter species, and may be the remnants of such negative selection. For example, the expansin gene (CMN_Ps00045; pseudo) and a metallophosphatase gene (CMN_Ps00028; pseudo) both are knocked out by premature stop codons consistently in all six C. nebraskensis genomes.

### I haven't looked into the genome content comparisons involving CIC292, as it carries a plasmid pCN1, which makes the process slightly less straightforward. I also took a look at Ryan's poster again, and found that the experiment in Fig. 3B very fascinating. Are the virulent isolate CIC058 and avirulent isolate CIC318 used in his experiment related to the sequenced isolates? I think the result of this experiment echos my hypothesis in 4).

## Email correspondence from [[August 16th, 2019]]
### CIC396 was chosen for genome comparison because Ryan described it initially as avirulent (see initial description in strain information sheet in file CMN genome strain info inplant cfu.xlsx).  We thought it would make a good comparison because we assumed it was an attenuated mutant of the original CIC251 (~CIC395).  From looking at his APS 2017 poster it appears that CIC396 is still virulent, although slightly less so than CIC395? (See APS Cmn Genome 2017 RRM_CI_RRM_dm_CI.pptx). I haven’t been able to find the disease severity ratings that are included in the poster, but have found pictures of leaf blades that look similar to the ones in the poster.  Haven’t found the descriptions to go with the photos, so am relying on those in the poster figure 1.  I’ll keep looking for photo labell

### I figured out the genome sequencing data designation A01 to F01 should most likely corresponds to CIC292, CIC313, CIC326, CIC328, CIC395 and CIC396, respectively, based on the sequence best match with Ryan's previously assemblies, expected genomes size and presence (or absence) of plasmids.

### The genome assemblies and annotation for the five C. nebraskensis isolates have been finished. I will write down the summary of the entire workflow and upload it into the shared Google Drive later. All five genomes (chromosomes) showed very high syntency with the reference genome for the type strain NCPPB 2581 (Mauve alignment image attached). I left out CIC328, as it turned out to be a Clavibacter-related strain other than C. nebraskensis. From the pilot run of the genome content analysis (chromosome only), I noticed that all five C. nebraskensis encode very similar number of genes. Relative to the mildly virulent CIC292/A01, nonvirulent CIC313/B01 and CIC326/C03, the highly virulent stains CIC395/E01 and CIC396/F01 only encode one additional ATPase gene, while both lost around ten genes. The nonvirulent CIC313/B01 appears to have the highest gene count. I will take a close look at those differences. At this point, I conjecture that high virulence of stains CIC395/E01 and CIC396/F01 is likely the consequence of gene loss which help the bacteria evade the plant immune system detection and thrive in host tissue without provoking strong immune responses, rather than the contrasting scenario that they might carry additional virulence genes to actively hijack/dampen plant immune responses. On the other hand, the additional genes specifically present in the nonvirulent isolates may elicit plant immune responses, although the possibility that genes specifically absent in the nonvirulent isolates may be involved in dampening plant immune responses could not be ruled out.

### Is there virulence assay data available for the type strain NCPPB 2581 tested on the same set of maize lines? If so, then I can include that strain into the comparative analysis as well. Please let me know what other analyses you would like to conduct, and your thoughts of how to put these together into a whole story for publication eventually. 

### ![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2FQualifying_Exam%2Fat-twmLSAR.jpg?alt=media&token=19013866-ed12-49ec-b8cc-1b8dde7eaf51)
